src/models.py

```diff
@@ -13,81 +11,7 @@
-
-    __tablename__ = "users"
-
-    id: int = Column(Integer, primary_key=True)  # noqa: A003
-    username: str = Column(String, nullable=False, unique=True)
-    password_hash: str = Column(String(64), nullable=False)
-
-    def _get_password_hash(self, password: str) -> str:  # pylint: disable=no-self-use
-        return hashlib.sha256(password.encode()).hexdigest()
-
-    def set_password(self, password: str) -> None:
-        """Use this method to set hashed password to User."""
-        self.password_hash = self._get_password_hash(password)
-
-    def check_password(self, password_to_check: str) -> bool:
-        """Use this method to check User's password."""
-        return self._get_password_hash(password_to_check) == self.password_hash
-
-    @classmethod
-    def create_user(cls, username: str, password: str) -> None:
-        """Use this method to create a new user."""
-        new_user = cls(username=username, password_hash=hashlib.sha256(password.encode()).hexdigest())
-        session = session_var.get()
-        session.add(new_user)
-        session.commit()
-
-    def create_session(self) -> UserSession:
-        """Use this method to create a new session."""
-        new_session = UserSession(session_id=str(uuid.uuid4()), user_id=self.id)
-        session = session_var.get()
-        session.add(new_session)
-        session.commit()
-        return new_session
-
-    @staticmethod
-    def get_users() -> list[User]:
-        """Use this method to get all users from users table."""
-        session = session_var.get()
-        return session.query(User).all()
-
-    @staticmethod
-    def get_user_by_credentials(username: str, password: str) -> User | None:
-        """Use this method to fetch a user from users table with a specific username and password."""
-        session = session_var.get()
-        user_to_fetch: User | None = session.query(User).filter_by(username=username).first()
-        if user_to_fetch and user_to_fetch.check_password(password):
-            return user_to_fetch
-        return None
-
-
-class UserSession(Base):
-    """A model class for user_session table."""
-
-    __tablename__ = "user_session"
-
-    id: int = Column(Integer, primary_key=True)  # noqa: A003
-    session_id: str = Column(String, nullable=False, unique=True)
-    user_id: int = Column(Integer, ForeignKey("users.id"), nullable=False)
-    user: User = relationship("User", uselist=False)
-
-    def delete(self) -> None:
-        """Use this method to delete a user session."""
-        session = session_var.get()
-        session.delete(self)
-        session.commit()
-
-    @staticmethod
-    def get_user_session_by_session_id(session_id: str) -> UserSession | None:
-        """Use this method to get a user session with a certain id."""
-        session = session_var.get()
-        return session.query(UserSession).filter_by(session_id=session_id).first()
+    from src.users.models import User  # noqa: TC004
```

Comment:
The reason why you have this `TC004` error is because we use old version of `flake8-type-checking` plugin. Actually this is a bug in `1.0.3` version of this plugin. I checked that out if we will upgrade its version, then problem is gone.

So you can update it to the latest version with this command:
```bash
pip install -U flake8-type-checking
```
and we need to update its version in our `requirements-dev.txt`.

After that you can remove this `# noqa` comment and flake should not complain about this error anymore.